src/canopy_cli/cli.py

```diff
@@ -580,8 +582,9 @@ def start(host: str, port: str, reload: bool,
         """
     )
 )
-@click.option("url", "--url", default="http://localhost:8000",
-              help="URL of the Canopy server to use. Defaults to http://localhost:8000")
+@click.option("url", "--url", default=f"http://localhost:8000/{API_VERSION}",
```

Comment:
Let's define a `DEFAULT_URL` at the top of the file?  
It suddenly makes much more sense

Reply:
done